.github/workflows/run-unit-tests.yml

```diff
@@ -33,3 +33,26 @@ jobs:
     - name: Test with pytest
       run: |
         pytest
+        
+  build-python-legacy:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6]
```

Comment:
Can we quickly estimate the dev-effort to upgrade from 3.6 (EOL circa 2021) to either 3.10 or 3.11?

ref: https://devguide.python.org/versions/#versions

Reply:
Yes, i have removed 3.6 and added 3.11 (3.10 was already existing).